src/public/views/ProfileView.js

```diff
@@ -22,19 +23,22 @@ export default class ProfileView extends View {
         this.insertElements = {};
         this._errorDiv = null;
         this.path = UPDATE_USER;
+        this.userInfo = {};
     }
     set infoAboutUser (userInfo) {
         this.userInfo = userInfo;
     }
     addInfo () {
-        this.insertElements.img.src = API_STATIC + this.userInfo.avatar;
+        console.log(this.userInfo);
+        this.insertElements.img.src = this.userInfo.avatar ? (this.userInfo.avatar) : DEFAULT_AVATAR;
```

Comment:
```suggestion
        this.insertElements.img.src = this.userInfo.avatar || DEFAULT_AVATAR;
```

Есть в JS особый хак с оператором `||`: он **возвращает** первое встретившееся значение, равное `true`, а если их нет, то последнее: https://learn.javascript.ru/logical-ops#znachenie-ili

Симметрично работает и оператор `&&` &ndash; возвращает первое ложное, а если их нет, то последнее значение.